cmd/otel-allocator/main.go

```diff
@@ -75,6 +86,7 @@ func main() {
 			}
 			os.Exit(0)
 		case event := <-watcher.Events:
+			events.WithLabelValues(allocatorWatcher.EventSourceToString[event.Source]).Inc()
 			switch event.Source {
```

Comment:
Would it be beneficial to have a metric for the time when we last received an event? Technically, we could see this in the most recent time when the count value changed but that seems complicated, especially if a lot of time has passed for ConfigMap.

Reply:
i think with the log already there and the ability to visualize the event from the delta of the counter that may be sufficient enough. Ideally we would be able to display some kind of event over the chart of this data.